controllers/openstackdataplanedeployment_controller.go

```diff
@@ -147,6 +147,21 @@ func (r *OpenStackDataPlaneDeploymentReconciler) Reconcile(ctx context.Context,
 		nodeSets.Items = append(nodeSets.Items, *nodeSetInstance)
 	}
 
+    // Add Ansible variables from the OpenStackDataPlaneDeployment to the AnsibleEESpec
+    ansibleEESpec.AnsibleVars = make(map[string]string)
+    for k, v := range instance.Spec.AnsibleVars {
```

Comment:
those should be set after nodeset ones, otherwise nodeset is overwriting the deployment vars